sys/shell_menu/Menu.cpp

```diff
@@ -83,7 +79,7 @@ void Menu::enter(void)
         i++;
     }
 
-    COGIP_DEBUG_COUT("Enter shell menu: " << current_menu->name().c_str());
+    COGIP_DEBUG_COUT("Enter shell menu: " << current_menu->name() << std::endl);
```

Comment:
This adds an other new line. is this wanted or an oversight to fix my previous PR error around this ?

Reply:
I did not see that COGIP_DEBUG_COUT adds a new line. Actually it should not, I missed that in your previous PR...

Reply:
For example, all functions printing JSON does not work anymore...

Reply:
I have removed the extra endl here for now, but this will have to be handled properly soon.

Reply:
> For example, all functions printing JSON does not work anymore...

Can you create an issue about this ?

Reply:
https://github.com/cogip/mcu-firmware/issues/77